btk/draw_blends.py

```diff
@@ -170,6 +172,8 @@ def __init__(
             channels_last (bool): Whether to return images as numpy arrays with the channel
                                 (band) dimension as the last dimension or before the pixels
                                 dimensions (default).
+            save_path (str): Path to save the results, ending by the file name root. If left
+                            as None, results will not be saved.
```

Comment:
I think now this description is outdated? See below for suggestion

```suggestion
            save_path (str): Path to a directory where results will be saved. If left
                            as None, results will not be saved.
```